sdk/js-query/src/query/ethCall.test.ts

```diff
@@ -0,0 +1,287 @@
+import {
+  afterAll,
+  beforeAll,
+  describe,
+  expect,
+  jest,
+  test,
+} from "@jest/globals";
+import Web3, { ETH_DATA_FORMAT } from "web3";
+import axios from "axios";
+import * as elliptic from "elliptic";
+import {
+  EthCallData,
+  EthCallQueryRequest,
+  PerChainQueryRequest,
+  QueryRequest,
+  QueryResponse,
+  sign,
+} from "..";
+
+jest.setTimeout(60000);
+
+const CI = false;
+const ENV = "DEVNET";
+const ETH_NODE_URL = CI ? "ws://eth-devnet:8545" : "ws://localhost:8545";
+
+const CCQ_SERVER_URL = "http://localhost:6069/v1";
+const QUERY_URL = CCQ_SERVER_URL + "/query";
+const HEALTH_URL = CCQ_SERVER_URL + "/health";
+const PRIVATE_KEY =
```

Comment:
Is this anything that we need to *really* keep private?

Reply:
Nope, it's the devnet guardian secret